src/plugins/embeddable/public/lib/panel/panel_header/panel_header.tsx

```diff
@@ -155,6 +157,45 @@ export function PanelHeader({
     );
   }
 
+  const getPlaceholderTitle = () => {
```

Comment:
Nit: I wonder if getTitle and getPlaceholderTitle could be combined. Seems to me like the only differences are the screen reader message, the title, and the class. Not super important though!